includes/breadcrumb.inc

```diff
@@ -98,10 +99,15 @@ EOQ;
     $filter_map = function ($filter) {
       return "FILTER($filter)";
     };
-    $query_filters[] = "sameTerm(?collection_predicate, <fedora-rels-ext:isMemberOfCollection>) || sameTerm(?collection_predicate, <fedora-rels-ext:isMemberOf>)";
+    $compound_object_predicate = variable_get('islandora_compound_object_relationship', 'isConstituentOf');
```

Comment:
Hi, so we should suggest a hook for this task in Islandra_compound_object, or the other way? Should every solution pack be able to define it's own brad crumb system based on this own rules? both ways  would also allow other modules to define own relationship predicates and functionality. But In terms of access time, getting this external variable is faster(optimal) than calling a hook, specially if this function is recursive, but in terms of code independence, i agree with you.